index.html

```diff
@@ -76,7 +76,7 @@ <h2 class="first">I'm Recillah <br> Glad To See You!</h2>
 
         <!-- Middle section -->
         <section class="works" id="portfolio">
-            <div class="single-card card1">
+            <!-- <div class="single-card card1 ">
```

Comment:
Great job team 🤝

- [ ] Kindly remove commented code from your HTML file. This is not regarded as a good practice.